### PR TITLE
Feature/mergeintegration

### PR DESCRIPTION
**Pull request by tsr-thulio**
Branch cointaining the front-end development merged with the DAO and API features.

### CODE REVIEW

**Comment by larruda** [comment]
Please update your branch with the latest changes from "develop" so this is mergeable.

**Comment by tsr-thulio** [comment]
merge with the develop done.

File: pom.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>com.googlecode.objectify</groupId>
             <artifactId>objectify</artifactId>
-            <version>5.1.6</version>
+             <version>5.1.6</version>
```
Comment (larruda) [inline pom.xml]: Please fix indentation on this file.

Comment (tsr-thulio) [inline pom.xml]: identation fixed

**Pull request merged**